src/main/java/meep/tool/Command.java

```diff
@@ -0,0 +1,306 @@
+package meep.tool;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import meep.ui.Ui;
+
+/**
+ * Central command handler for Meep. Contains static methods that mutate the
+ * in-memory message and
+ * task lists and print responses via {@link meep.ui.Ui}.
+ */
+class Command {
+    private static MessageList messages = new MessageList();
+    private static TaskList tasklist = new TaskList();
+
+    /**
+     * Adds a raw input message to the internal message list.
+     *
+     * @param message user input
+     * @return true if the operation completes
+     */
+    public static boolean addMessage(String message) {
+        messages.addMessage(message);
+        return true;
+    }
+
+    /**
+     * Prints a hello response.
+     *
+     * @return true always
+     */
+    public static boolean helloCommand() {
+        Ui.printResponse("Hello there!");
+        return true;
+    }
+
+    /**
+     * Prints a canned "how are you" response.
+     *
+     * @return true always
+     */
+    public static boolean howAreYouCommand() {
```

Comment:
I like the idea of this very much, very creative and funny, but I think the naming should be a verb since this is a function
```suggestion
    public static boolean replyToHowAreYouCommand() {
```